cmd/gaia/app/genesis.go

```diff
@@ -114,34 +115,34 @@ func GaiaAppGenState(cdc *codec.Codec, appGenTxs []json.RawMessage) (genesisStat
 			err = errors.New("must provide genesis StdTx with exactly 1 CreateValidator message")
 			return
 		}
-		msg := msgs[0].(stake.MsgCreateValidator)
+		_ = msgs[0].(stake.MsgCreateValidator)
```

Comment:
What's this for? Are we just doing a type assertion? If so, should we just check the type and return an error if it's not the right type?

Reply:
Done, thanks